# Computational Methods for Epitranscriptomics 2024

<img src="https://github.com/F0nz0/EpitranscriptomicsCourse2024/blob/main/epitrascriptomics_bari.png"></img>

<strong>Organisers:</strong><br>
<a href="mailto:ernesto.picardi@uniba.it ">Ernesto Picardi (UNIBA, IT)</a><br>
<a href="mailto:f.deleo@ibiom.cnr.it">Frencesca De Leo (CNR-IBIOM, IT)</a><br>
<a href="mailto:francesco.nicassio@iit.it ">Francesco Nicassio (IIT, IT)</a><br>
<a href="mailto: ">Anna Marabotti (UNISA, IT)</a><br>
<a href="mailto:graziano.pesole@uniba.it ">Graziano Pesole (UNIBA, IT)</a><br>

<br>
<strong>Instructors:</strong><br>
<a href="mailto:adriano.fonzino@uniba.it "> Adriano Fonzino (UNIBA, IT)</a><br>
<a href="mailto:asilvestris@alice.it "> Alessandro Silvestris (UNIBA, IT)</a><br>
<a href="mailto: francesco.nicassio@iit.it">Francesco Nicassio (IIT, IT)</a><br>
<a href="mailto:ernesto.picardi@uniba.it ">Ernesto Picardi (UNIBA, IT)</a><br>
<a href="mailto: tleonardi@gmail.com">Tommaso Leonardi</a><br>
<a href="mailto:camilla.ugolini@iit.it ">Camilla Ugolini (IIT, IT)</a><br>
<a href="mailto:logan.mulroney@iit.it ">Logan Nulroney (IIT, IT)</a><br>

<br>
<strong>Helpers:</strong><br>
<a href="mailto: ">Chiara Bruno (CNR, IT)</a><br>
<a href="mailto: ">Marilena D'Ambrosio (CNR, IT)</a><br>
<a href="mailto:pietro.daddabbo@uniba.it ">Pietro Dâ€™Addabbo (UNIBA, IT)</a><br>
<br><br>


<strong>Course Description:</strong><br>
Epitranscriptome refers to all chemical modifications affecting cellular RNAs. Although more than 170 different types have been identified up to now, their precise detection remains challenging. Profiling epitranscriptome modifications require ad hoc sequencing technologies and dedicated computational workflows. While short-read RNA sequencing (e.g., Illumina) is the de facto method for unveiling RNA editing events (rather than transient modifications), direct RNA Sequencing by Oxford Nanopore Technology has recently become one of the most powerful techniques to profile the full repertoire of RNA modifications. Its capacity to directly sequence full-length, native RNA molecules without the need for reverse transcription or PCR amplification offers remarkable advantages compared to other short-read sequencing technologies. However, it also presents various experimental and analytical challenges.

This training course will provide a foundation on the experimental planning, analytical strategies, and computational approaches for applying Nanopore direct RNA Sequencing to profile RNA modifications. Through a combination of theoretical lectures and practical exercises, participants will learn the fundamental concepts and bioinformatics skills that will allow them to start using Nanopore sequencing for epitranscriptome analysis. Additionally, the course will also cover theoretical aspects of RNA editing as well as practical sessions focused on its detection in human/mouse transcriptomes by Illumina RNAseq data. Participants will acquire specific skills in command-line tools (REDItools) to call, annotate (REDIportal), and filter RNA variants.

You can find <a href="https://docs.google.com/document/d/1dxu3OFrOvllSAV36AZqIyslyePhazyST/edit">here</a> the complete program.
